pax/content.py

```diff
@@ -65,16 +65,24 @@ class ContentClient(object):
     Content client.
     """
     def __init__(self,axilent_connection):
-        self.content_resource = axilent_connection.resource_client('axilent.content','content')
+        self.__content_resource = None
```

Comment:
I actually didn't know about the leading double-underscore name mangling feature in Python until now when I looked up why you might do that.  Interesting stuff.